public/admin/config.yml

```diff
@@ -726,3 +726,59 @@ collections:
                 name: vaccinationSignupUrl
                 required: false
                 widget: string
+  - label: Vaccination Phases & Alerts
+    name: vaccinationPhasesAndAlerts
+    label_singular: Vaccination Phase
+    description: Details on vaccination phases and alerts.
+    format: json
+    files:
+      - name: Vaccine Information-State
+        label: State Vaccine Information 
+        file: src/cms-content/vaccines/state-vaccine-phases.json
+        fields:
+          - label: Regions
+            name: regions
+            widget: list
+            fields:
+              - label: Location Name
+                name: locationName
+                widget: string
+              - label: Fips Code
+                name: fips
+                widget: string
+              - label: Hidden
+                name: hidden
+                widget: boolean
+                required: false
+              - label: Eligibility Information URL
+                name: eligibilityInfoUrl
+                widget: string
+              - label: Email alert version
+                name: emailAlertVersion
+                default: 0
+                widget: number
+              - label: Phases
+                name: eligibilityPhases
+                widget: list
+                collapsed: true
+                fields:
+                  - label: Group Phase
+                    name: groupPhase
+                    widget: string
+                  - label: Group Tier
+                    name: groupTier
+                    widget: string
+                  - label: Group Description
+                    name: groupDescription
+                    widget: text
+                  - label: Expanded Definition URL
+                    name: expandedDefinitionUrl
+                    required: false
+                    widget: string
+                  - label: Eligibility Start Date
+                    name: eligibilityStartDate
+                    widget: date
+                  - label: Updated at
+                    name: updatedAt
```

Comment:
This should probably be date only

Reply:
why? seems like the time could be helpful.  this feels like an internal field